libs/language-server/src/test/assets/block-definition/invalid-block-as-multiple-pipe-inputs.jv

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+pipeline Test {
+  block TestPipeTo oftype TestTableInput {
+  }
+
+  block TestPipeFrom oftype TestFileOutput {
+  }
+
+  pipe {
+    from: TestPipeFrom;
+    to: TestPipeTo;
+  }
+
+  pipe {
+    from: TestPipeFrom;
+    to: TestPipeTo;
+  }
+}
```

Comment:
I'd recommend to overthink the naming. I'd go with something like this:
```suggestion
pipeline Pipeline {
  block BlockTo oftype TestTableLoader {
  }

  block BlockFrom oftype TestFileExtractor {
  }

  pipe {
    from: BlockFrom;
    to: BlockTo;
  }

  pipe {
    from: BlockFrom;
    to: BlockTo;
  }
}
```

For me, calling blocks `Pipe` seems pretty confusing. Regarding the names of the test block types, it would also be helpful to stick to the established conventions (i.e. `Extractor` for block types with no input and `Loader` for block types with no output).

To make the test more interesting, consider using the arrow pipe syntax for one of the pipes (`A -> B`).

Reply:
I have separate tests for chained vs single pipes so I did not deem that necessary.
However I did decide to switch all of these (where possible) to chained pipes to further compact files.